#!/usr/bin/env python
PACKAGE = "template_package"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("int_param",         int_t,    0, "Integer Value", 127,   0, 255)
gen.add("double_param",    double_t,   0, "Double Value",  0.5, 0.0, 1.0)

exit(gen.generate(PACKAGE, "template_package", "PublishNodeDynCfg"))
